charts/incubator/cryptpad/Chart.yaml

```
@@ -1,9 +1,9 @@
 ---
 apiVersion: v2
-appVersion: v4.12.0-nginx
+appVersion: v4.14.1-nginx
 description: cryptpad helm package
 name: cryptpad
-version: 0.3.2
+version: 0.4.0
```

```suggestion
version: 0.4.1
```

I think a patch version increment can suffice for version bumps.

---

I'm used to sync the bump type, so if the app bumps the major, the chart bumps the major.
But I'm definitely fine with following your convention :)